### PR TITLE
[Doc] update the example of explain analyze statement doc

### LINKED ISSUE #9419: doc about explain analyze is incorrect
explain analyze statement do not show estimates after fix #9396 .

### PR DESCRIPTION
**Pull request by kyo-tom**
Hi,I find some example result is different with the latest version of trino，I expect we can update. Fixes #9419 
Here is update:
- [ ] update the examples of explain analyze [verbose] statement

### CODE REVIEW

File: docs/src/main/sphinx/sql/explain-analyze.rst [hunk 1]
```diff
@@ -83,20 +99,58 @@ For example, the window function operator will output the following::
...
+            └─ Window[]
+               │   Layout: [clerk:varchar(15), count:bigint]
+               │   CPU: 38.00ms (3.43%), Scheduled: 54.00ms (1.09%), Output: 8130 rows (230.24kB)
+               │   Input avg.: 8130.00 rows, Input std.dev.: 0.00%
+               │   Active Drivers: [ 1 / 1 ]
+               │   Index size: std.dev.: 0.00 bytes, 0.00 rows
+               │   Index count per driver: std.dev.: 0.00
+               │   Rows per driver: std.dev.: 0.00
+               │   Size of partition: std.dev.: 0.00
+               │   count := count("clerk") RANGE UNBOUNDED_PRECEDING CURRENT_ROW
```
Comment (ebyhr) [inline docs/src/main/sphinx/sql/explain-analyze.rst:128]: Leave only L119-L128 as the previous plan because we want to show `Window[]` example here.

**Review by ebyhr — COMMENTED** [review]
Could you use explain result using `sf1` schema instead of `tiny`?

Also, please change the commit title as `Update example of EXPLAIN ANALYZE documentation`.
https://github.com/trinodb/trino/blob/master/.github/DEVELOPMENT.md#format-git-commit-messages

File: docs/src/main/sphinx/sql/explain-analyze.rst (hunk 1 above)
Comment (ebyhr) [reply docs/src/main/sphinx/sql/explain-analyze.rst:128]: > ... tpch:orders:sf0.01 ...

The above line implies you used `tiny` schema. Please use `sf1` and show only `Window[]` (L119-L128 ).

**Comment by mosabua** [comment]
:wave: @kyo-tom - this PR has become inactive. If you're still interested in working on it, please let us know. Specifically you probably have to rerun the queries as mentioned by @ebyhr and also use the latest version of Trino. 

We're working on closing out old and inactive PRs, so if you're too busy or this has too many merge conflicts to be worth picking back up, we'll be making another pass to close it out in a few weeks.

**Comment by colebow** [comment]
Closing this one out due to inactivity, but please reopen if you would like to pick this back up.

**Pull request closed without merging**